## Martini Tutorials
### 1. Protein 

>  The first layer contains the mapping and chosen chemical bead types as well as the bonded terms; the second layer is the structure bias model.the first layer corresponds to the standard definition of any molecule in the Martini universe, the second layer is more specific for proteins.

>  To maintain the secondary, tertiary and quaternary structure of proteins, the directionality of interactions, in particular hydrogen bonds, are crucial. Due to the spherical potentials used in Martini 3 and many other CG force fields, this directionality is lost. Therefore, a structure bias model is required to stabilize protein structures. 

>  The first layer of the current Martini 3 protein model is mostly inherited from the Martini 2 model with moderate updates concerning the new mapping guidelines, bead types, and sizes.The Martini 3 mappings and bead types of the proteinogenic amino acids are depicted in Figure 1. A key difference to the previous model is that the bead type of the backbone (BB) bead does not depend on the secondary structure anymore but is represented by a P2 bead. Exceptions are the special cases glycine (SP1), alanine (SP2), valine (SP2), proline (SP2a), and terminal beads (Q5 for charged and P6 for neutral termini).

><img width="6519" height="5520" alt="Image" src="https://github.com/user-attachments/assets/d18bdcc4-e3a3-412c-86c4-4e0c57e38dea" /> Figure 1: Mapping and chemical bead types of amino acids in the Martini 3 protein model. The main classes of chemical bead types are indicated by color: P (polar, in red), N (intermediately polar, in blue), C (nonpolar, in gray), and Q (charged, in lime). Different bead sizes are indicated by the radius of the bead, with regular (no symbol), small (S), and tiny (T) beads.
- 对于second layer需要选择3种中的一种来保持蛋白质的二级，三级，四级结构。
> The Martini protein models require specific bonded terms to model the secondary structure, which is in most cases combined with side-chain dihedral corrections(二面角).There are three main options for structural bias models in layer 2 available, typically used to stabilize tertiary and quaternary structures, namely Elastic Network (EN), GōMartini, and OLIVES. Whereas the ad-hoc EN approach provides the most robust and straightforward way of stabilizing a protein structure, and has traditionally been used in Martini, the bioinformatics-based GōMartini approach is nowadays the method of choice as it has proven to be a more versatile method, striking a balance between protein stability and flexibility. Most recently, OLIVES was introduced as a physics-based variant of GōMartini, with the particular prospect of biasing quaternary structures as well.
- 3种model是基于all-atom模拟获得的参数
> In their simplest form, each of these three structure bias models relies on an atomistic reference structure which can be obtained from experiments or prediction software. The EN model uses harmonic potentials between the BB beads within a cutoff distance to maintain the protein structure. In contrast, both the GōMartini and OLIVES models employ Lennard-Jones potentials in a Gō-type manner, allowing for contact dissociation. In the GōMartini model, the potentials are defined from a contact map based on native contacts, evaluated through overlap and restricted contacts of structural units [18, 19], combined with a distance cutoff. The OLIVES model, on the other hand, defines its bias by evaluating the hydrogen bonding interactions between BB as well as side chain (SC) beads [17]. For small structured systems, such as single α-helices, the waiver of a structural bias model can be an option as well.
### 1.1 Setup
- 研究对象：4JJR蛋白质也叫the protein kinase蛋白激酶中的casein kinase delta 1 (CK1δ)酪蛋白激酶1δ；蛋白质是由多个氨基酸脱水缩合而形成的链状化合物，每一个amino acid由于侧链不同（可能还有别的不同比如bonds/angles）可以被分成约20种，用英文字母命名，也叫残基residues，蛋白质的一级结构就是表示残基数量和排序序列的；二级结构和三级结构都用于表示蛋白质长链的conformation构象，二级结构描述的是局部片段的折叠情况，一般分为三种情况，α-螺旋（alpha helix），β-折叠/β-片层（beta sheet），转角/环（turn/loop）；三级结构描述的是一整条多肽链如何构象折成一个三维形状，这个三维链条也叫单体monomer；四级结构是描述多个折好的单体怎么组装成复合体complex，两条chain链组成的就叫dimer，三条链组成的就是trimer。我们研究的4JJR就是由chainA+chainB组成的dimer。
> In this tutorial, we will prepare different Martini 3 structure bias models for the protein kinase casein kinase delta 1 (CK1δ).we will focus solely on the structured region of this protein (residues 1-292), while the IDR (residues 293-415) will be covered in next part. We use the experimental crystal structure with PDB code 4JJR as a starting structure.
-   record type；原子序号从1开始数atom serial number ；原子名称atom name；残基名 residue name这里是亮氨酸 Leucine；chain identifier链序号，这里是A链；残基序号residue sequence number这是 A 链的第 3 号残基，从3号开始说明1，2号残基可能缺失；三维坐标x,y,z coordinates;占有率occupancy，实验结构里表示该位置出现该构象的比例；对理论模型一般常设成 1.00；local score/local quality estimater，通常 < 0.6 的残基被认为质量较低；元素符号element
> ATOM 1 N LEU A 3 1.075 44.827 7.365 1.00 0.66 N
- 先把.pdb文件作为topo文件进行all-atom simulation作为CG 模拟的reference(Atomistic Reference Simulation)
> We recommend running atomistic reference simulations to validate the CG Martini 3 protein model. For example, you can use the [CHARMM-GUI Solution Builder](https://charmm-gui.org/?doc=input/solution) to build the system and generate topologies using the final PDB file generated in Section I.I.1. Solvate the system using 0.15 M of NaCl (the same salt concentration that will be used for the CG simulation later) and neutralize it by adding the appropriate number of counter-ions. Here, we recommend conducting at least two replicas of 1 μs each.
-使用martinize2进行beads mapping 但只是unbias的第一层，后续还需要继续用三种bias model改进
> Now that we have a complete atomistic starting structure of CK1δ, we can prepare the CG structure. Here, we use the [Martinize2](https://github.com/marrink-lab/vermouth-martinize) program (version 0.12.0), which generates a CG structure and the necessary topologies to perform simulations with GROMACS based on an atomistic reference structure.In general, Martinize2 assigns placeholder names to molecules and topology files.
- 